app/src/main/java/espertolabs/esperto_ble_watch/ScanActivity.java

```diff
@@ -126,9 +126,9 @@ public void createAccounts(String [] userData, String deviceAddress){
         newUser.setUsername(userData[2]);
         newUser.setPassword(userData[3]);
         newUser.setDeviceAddress(deviceAddress);
-        newUser.setGoalSetting(userData[4]);
-        newUser.setFirstName(userData[0]);
-        newUser.setLastName(userData[1]);
+        //newUser.setGoalSetting(userData[4]);
+        //newUser.setFirstName(userData[0]);
+        //newUser.setLastName(userData[1]);
         //make asynchronous method call to synchronous DynamoDB
         // TODO: change to API gateway
```

Comment:
Does this still need to be updated to API gateway?

Reply:
I'll modify this once I write the cognito code to sign up